docusaurus/docs-internals/editor-controls/multivalue-controls/iconsPicker.md

```diff
@@ -0,0 +1,271 @@
+---
+toc_max_heading_level: 4
+---
+
+# Icons Picker
+
+The `iconsPicker` control in Brizy presents a selection of icons. This control allows users to choose icons for various purposes. The blue checkmarks provide visual feedback on which icons are currently active or selected.
```

Comment:
review description because this control let us to select not only icons, this control usually is an checkbox